src/components/Monaco/ReactMonacoEditor.tsx

```diff
@@ -71,7 +82,7 @@ export function ReactMonacoEditor({
           ...props.options,
           wordWrap: toolbarState.wordWrap,
         }}
-        onMount={(editor) => setEditor(editor)}
```

Comment:
This was overriding the `onMount` prop declared in the `CodeEditor` component:

https://github.com/grafana/k6-studio/blob/0678c9b13d4a2d80ea7327e25d1213c1cc9b07f1/src/components/Monaco/CodeEditor.tsx#L20-L40

This prevented the content of script preview from being set.